src/content/docs/infrastructure/prometheus-integrations/install-configure-prometheus-agent/install-prometheus-agent.mdx

```diff
@@ -0,0 +1,181 @@
+---
+title: Install and configure the Prometheus agent alongside the Kubernetes integration
+tags:
+  - Integrations
+  - Prometheus integrations
+  - Install and configure Prometheus agent
+metaDescription: 'How to install, update, or uninstall your New Relic Prometheus agent alongside the Kubernetes integration.'
+---
+
+import infrastructurePrometheusIoDashboard from 'images/infrastructure_screenshot-crop_prometheus-io-dashboard.png'
+
+import infrastructurePrometheusNrqlQuery from 'images/infrastructure_screenshot-crop_prometheus-nrql-query.png'
+
+New Relic's Prometheus agent is a thin wrapper on top of Prometheus server to run it in agent mode. With this integration, you can create a YAML configuration file used by Prometheus that's ready to send metrics to New Relic.
+
+Learn more about the solution from [its repository](https://github.com/newrelic/newrelic-prometheus-configurator/blob/main/charts/newrelic-prometheus-agent/values.yaml#L184).
```

Comment:
is the link `https://github.com/newrelic/newrelic-prometheus-configurator/blob/main/charts/newrelic-prometheus-agent/values.yaml#L184` correct?

Reply:
I think not, instead if should point to the main page of the repo